_pages/ato/levels.md

```diff
@@ -5,15 +5,17 @@ navtitle: Impact Levels
 
 ### Overview
 
-Categorization assesses the functionality of the system and the data it contains. The methodology used by federal agencies assess the impact on the public or government if the `confidentiality`, `integrity`, or `availability` of the system is lost. These three security objectives are assigned one of three impact levels: `low`, `moderate`, or `high`. 
+The impact level of your project is very important, and affects the process you'll follow to launch your project. At first you'll be making a prediction of your project impact level. As you enter the ATO assessment process the impact level will be determined with the help of your Infrastructure Lead and AO.
```

Comment:
Agreed, we should combine or at least align the language here with the language elsewhere that says to determine level with your Infrastructure Lead.